### PR TITLE
hserver framework submit

### PR DESCRIPTION
**Pull request by heixiaoma**

### CODE REVIEW

**Review by heixiaoma — COMMENTED** [review]
ok

File: frameworks/Java/hserver/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# HServer Benchmarking Test
```
Comment (heixiaoma) [inline frameworks/Java/hserver/README.md:1]: ok

File: frameworks/Java/hserver/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+This is the HServer portion of a [benchmarking test suite](../) comparing a variety of web development platforms.
```
Comment (heixiaoma) [inline frameworks/Java/hserver/README.md:2]: ok

File: frameworks/Java/hserver/src/main/java/com/test/hserver/controller/TestController.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ * @author hxm
+ */
+@Controller
+public class TestController {
+
+    private static Message message = new Message();
+
+    @GET("/json")
+    public Message json() {
+        return message;
```
Comment (nbrady-techempower) [inline frameworks/Java/hserver/src/main/java/com/test/hserver/controller/TestController.java:17]: You'll need to instantiate `Message` on every request, so please move `new Message()` to here.

Comment (heixiaoma) [reply frameworks/Java/hserver/src/main/java/com/test/hserver/controller/TestController.java]: OK, thank you

**Comment by heixiaoma** [comment]
Hello, the inspection has passed. What else do I need to do? Or just wait for the merger?

**Pull request merged**